mrt/manager.py

```diff
@@ -210,3 +209,26 @@ def remove_missing_countries(ctx):
                 participant.represented_country = None
 
         db.session.commit()
+
+@cli.command()
+@click.pass_context
+def add_verified_flag_mp(ctx):
+    app = ctx.obj['app']
+    with app.test_request_context():
+        for meeting in Meeting.query.all():
+            if meeting.settings.get('media_participant_enabled', False):
+                if meeting.custom_fields.filter_by(
+                            custom_field_type = unicode(CustomField.MEDIA),
+                            slug = 'verified').all() != []:
+                    continue
+                cf = CustomField(slug = 'verified',
+                                meeting_id = meeting.id,
+                                field_type = CustomField.CHECKBOX,
+                                is_primary = True,
+                                custom_field_type = CustomField.MEDIA)
```

Comment:
You shouldn't use spaces around `=` sign when used to indicate a keyword argument or a default parameter value. More on whitespace [here](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).